Resources/Prototypes/Datasets/Names/pitbull.yml

```diff
@@ -0,0 +1,34 @@
+- type: dataset
+  id: names_pitbull
+  values:
+  - Zeus
+  - Magnum
+  - Diesel
+  - Satan
+  - Dante
+  - Midas
+  - Blue
+  - Petunia
+  - Luna
+  - Bubbles
+  - Rainbow
+  - Echo
+  - Cuddles
+  - Killer
+  - Paisley
+  - Echo
+  - Sweetie
+  - Tutu
+  - Reaper
+  - Peaches
+  - Cujo
+  - Cuddles
```

Comment:
Cuddles is in here twice

Reply:
Cuddles is a good enough name to be seeded to appear more